src/js/providers/html5.js

```diff
@@ -1145,7 +1150,11 @@ define([
             }
             _addTracksListener(tracks, 'change', _textTrackChangeHandler);
             if (_textTracks && _textTracks.length) {
-                _this.trigger('subtitlesTracks', { tracks: _textTracks });
+                var subtitlesTracks =  { tracks: _textTracks };
+                if (_defaultTextTrackIndex > -1) {
```

Comment:
This if statement is not needed, since if `_defaultTextTrackIndex` is `-1` then  `defaultIndex` will be set to `0`. This is the default value in captions.js > `_selectDefaultIndex`.